chain-abci/src/main.rs

```diff
@@ -25,10 +29,26 @@ pub struct Config {
     host: String,
     genesis_app_hash: String,
     chain_id: String,
-    enclave_server: Option<String>,
     tx_query: Option<String>,
+    // if different from `tx_query`
+    tx_query_listen: Option<String>,
     launch_ra_proxy: bool,
     remote_attestation: SpRaConfig,
+    data_bootstrap: TdbeConfig,
+}
+
+/// TODO: more concrete when ready
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub struct TdbeConfig {
+    /// Optional TM RPC address of another TDBE server from where to fetch data
+    /// TODO: it'll get txids + the TDBE connection details from RPC
+    /// TODO: DNS can be obtained from RPC?
+    pub remote_rpc_address: Option<String>,
```

Comment:
how about adding examples for these addresses?
like
127.0.0.1:3433  or
tcp://127.0.0.1:3433 ?